src/PostgrestFilterBuilder.ts

```diff
@@ -32,11 +32,6 @@ export default class PostgrestFilterBuilder<
   RelationName = unknown,
   Relationships = unknown
 > extends PostgrestTransformBuilder<Schema, Row, Result, RelationName, Relationships> {
-  eq<ColumnName extends string & keyof Row>(
-    column: ColumnName,
-    value: NonNullable<Row[ColumnName]>
-  ): this
-  eq<Value extends unknown>(column: string, value: NonNullable<Value>): this
```

Comment:
The only thing I'm curios about is if we ever allowed for `.eq<'number'>('id', 42)` for example 🤔